documentation/modules/installing-mtv-operator.adoc

```diff
@@ -17,7 +17,7 @@ endif::[]
 
 .Prerequisites
 
-* {ocp} {ocp-version} or later installed.
+* {ocp} {ocp-y-version} or later installed.
 * {virt} Operator installed on an OpenShift migration target cluster.
```

Comment:
Please remove "or later". We don't want to suggest that {ocp-version}+1 or later is supported.